ios/TrueSheetView.swift

```diff
@@ -275,6 +302,31 @@ class TrueSheetView: UIView, RCTInvalidating, TrueSheetViewControllerDelegate {
 
   // MARK: - Methods
 
+  private func initializeContent() {
+    guard let contentView, let footerView else {
+      return
+    }
+
+    // Update content containers
+    setupScrollable()
+
+    // Set initial content height
+    let contentHeight = contentView.bounds.height
+    setContentHeight(NSNumber(value: contentHeight))
+
+    // Set initial footer height
+    let footerHeight = footerView.bounds.height
+    setFooterHeight(NSNumber(value: footerHeight))
+
+    // Present sheet at initial index
+    let initialIndex = self.initialIndex.intValue
+    if initialIndex >= 0 {
+      present(at: initialIndex, promise: nil, animated: false)
```

Comment:
Can we add [animateOnMount](https://ui.gorhom.dev/components/bottom-sheet/props#animateonmount) 🙏 . In our usecase we mount the sheet on navigation so it needs to be animated. Can be also done in a separate PR to reduce this PR's scope!

Reply:
This pr includes `onReady` event which you can take advantage to present the sheet on mount.

Reply:
yeah that works as well. Thanks

Reply:
Even though onReady is neat, I think an animateOnMount prop would be best. It would save one JS roundtrip: Event -> Callback -> Present -> Bridge -> Native react.

Having a prop and instantly opening on native whenever the sheet is ready should be the fastest solution imho.

Reply:
That makes sense. Agreed 👍

Reply:
Thanks. If initialIndex is only used for animateOnMount, we could just have one prop instead of to. Either pass a number to animateOnMount or just use initialIndex imo.

Reply:
I would prefer two props to make it more flexible. I would imagine some user to disable animation for the initial index, like on a maps app. `animateInitialIndex` would actually be more appropriate name for it.

Reply:
Just nits, but I don't like the naming of `animateInitialIndex` :D
Anyway, thanks for your work